src/bindings/node/node.mli

```diff
@@ -48,6 +48,20 @@ module Stream : sig
   val end_ : t -> unit
 end
 
+module Path : sig
+  val delimiter : string
```

Comment:
Or better yet, we could try binding `delimiter` as a `char`.

Reply:
`char` is not a supported [gen_js_api type](https://github.com/LexiFi/gen_js_api/blob/master/TYPES.md#js-able-types). Do you mean I should move the assertion and indexing to the node module?

Reply:
I meant using `char` directly. But if it's not supported, then I suppose moving our cast to the bindings also makes sense.